service_actor.gemspec

```diff
@@ -43,8 +43,10 @@ Gem::Specification.new do |spec|
   # Debugger
   spec.add_development_dependency "pry"
 
-  # Linting
-  spec.add_development_dependency "rubocop"
+  # Linting rubocop-lts v12 is a rubocop wrapper for Ruby >= 2.4,
+  #   and should only be bumped when dropping old Ruby support
+  # see: https://dev.to/pboling/rubocop-lts-1e31
```

Comment:
If you're not a fan of `See: URL` type comments, let me know, and I'll remove it.

Reply:
Love these, great way to get more context!